packages/docs/core-concepts/actions.md

```diff
@@ -62,9 +62,9 @@ export default defineComponent({
 })
 ```
 
-## Accessing other stores actions
+## Accessing other stores' actions
 
-To use another store, you can directly _use it_ inside of the _action_:
+To use another store's action, you can directly _use it_ inside of the _action_:
```

Comment:
This one was correct

Reply:
Apologies.

Removed the change to the heading as well. The other headings in the docs are written the same way and I rather a PR to address them all. I'll attempt one if I find the time. Let's just do the really obvious typos in this one.